priv/perf/scripts/generate_api_client.sh

```diff
@@ -23,10 +23,9 @@ echo "Generating childchain clients..."
 echo "------------------------------------------------------"
 docker run --rm \
     -v ${PWD}/apps:/apps \
-    -v ${PWD}/../../apps/omg_child_chain_rpc/priv/swagger/:/swagger \
     --user $(id -u):$(id -g) \
     openapitools/openapi-generator-cli generate \
-    -i /swagger/operator_api_specs.yaml \
+    -i https://raw.githubusercontent.com/omgnetwork/grandfather/master/apps/omg_child_chain_rpc/priv/swagger/operator_api_specs.yaml?token=AATWQG6U2GJE3VHVMVDEDWK7NCGYE \
```

Comment:
I guess grandfather got rebranded 😛.
Also.....the token if that is sth should be confidential you might want to delete that token. (I guess it is GitHub API token?)

Reply:
it's just their tracking token, anyhow, its open now so I;ll switch to omg-childchain-v1!